src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -0,0 +1,31 @@
+__PROJECT_DIR=$1
+__TOOLS_DIR=$2
+__DOTNET_CMD=$3
+__TOOLRUNTIME_DIR=$__PROJECT_DIR/bin/obj/Unix.AnyCPU.Debug/ToolRuntime
+
+__BUILDERRORLEVEL=0
+
+if [ ! -d "$__PROJECT_DIR" ]; then
+   echo "ERROR: Cannot find project root path at '$__PROJECT_DIR'. Please pass in the source directory as the 1st parameter."
+   exit 1
+fi
+
+if [ ! -d "$__TOOLS_DIR" ]; then
+   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+   exit 1
+fi
+
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+   exit 1
+fi
+
+# Temporary hack to make dnu restore more reliable.
+export MONO_THREADS_PER_CPU=2000
```

Comment:
It's not needed ever. The dotnet CLI is using a CoreCLR based DNX so mono settings don't apply.